src/native/public/mono/metadata/object-forward.h

```diff
@@ -19,4 +19,9 @@ typedef struct _MonoException MONO_RT_MANAGED_ATTR MonoException;
 typedef struct _MonoReflectionAssembly MONO_RT_MANAGED_ATTR MonoReflectionAssembly;
 typedef struct _MonoReflectionTypeBuilder MONO_RT_MANAGED_ATTR MonoReflectionTypeBuilder;
 
+typedef struct _MonoVarianceSearchTableEntry {
+    int depth, offset;
```

Comment:
nit1: I'm not a huge fan of `,` when declaring members of a struct.

nit2: `MonoClass:idepth` is 16 bits. `depth` can be `uint16_t`